web/src/beta/lib/core/engines/Cesium/Sketch/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { Cartesian3, type PolygonHierarchy, type Scene } from "@cesium/engine";
```

Comment:
Some functions are defined in [here](https://github.com/reearth/reearth/blob/main/web/src/beta/lib/core/engines/Cesium/Feature/HeatMap/utils.ts) as well, so could you update it to define in common place like `Cesium/utils/polygon.ts` or something?

Reply:
Updated. Merged into `Cesium/utils/polygon.ts`.